includes/Blocks/CorePostTerms.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Core Post Terms Block
+ *
+ * @package WPGraphQL\ContentBlocks\Blocks
+ */
+
+namespace WPGraphQL\ContentBlocks\Blocks;
+
+use WPGraphQL\ContentBlocks\Registry\Registry;
+use WPGraphQL\Model\Term;
+use WP_Block_Type;
+
+/**
+ * Class CorePostTerms.
+ */
+class CorePostTerms extends Block {
+	/**
+	 * String fields for the block.
+	 */
+	public const STRING_FIELDS = [ 'prefix', 'suffix', 'term' ];
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function __construct( WP_Block_Type $block, Registry $block_registry ) {
+		parent::__construct( $block, $block_registry );
+
+		foreach ( self::STRING_FIELDS as $field ) {
```

Comment:
This seems like an unnecessary abstraction. I'd recommend using `register_graphql_fields()` (plural) directly

Reply:
@justlevine Good idea. 
I’ve updated the code to remove the extra method call and directly register fields inside `register_graphql_fields()`. The abstraction has been minimized, but `get_string_field_config()` is still used for maintainability